presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaSplit.java

```diff
@@ -42,9 +42,9 @@
     private final Optional<String> keyDataSchemaContents;
     private final Optional<String> messageDataSchemaContents;
     private final int partitionId;
-    private final long start;
```

Comment:
why change start from final to non-final? We should keep start and end untouched, is it?

Reply:
Fixed.